### ISSUE TITLE
Header yellow does not have enough contrast with blue background

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#22]
Unfortunately, even white text does not have enough contrast with the background, so we will need to make the blue darker. 

Depending on how hard it is to edit the background image, we may want to just replace the background image with a css gradient.
